plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -279,6 +279,7 @@
 import static io.trino.plugin.hive.metastore.SemiTransactionalHiveMetastore.cleanExtraOutputFiles;
```

Comment:
nit: you don't need to make commit messages so narrow, using upto 80 chars per line is okay

Reply:
Yeah, you're right I forgot the body can be wider, but isn't it supposed to be 72 for the body of the message? I take this from https://cbea.ms/git-commit/#wrap-72

Reply:
Typically we stick to 72 chars for the title to avoid wrapping titles in GH PRs.
For the commit body we keep it loosely around 80 chars.